src/main/java/com/nakaligoba/backend/controller/MemberController.java

```diff
@@ -34,6 +39,17 @@ public ResponseEntity<SignupResponse> signup(@Valid @RequestBody SignupRequest r
         return ResponseEntity.ok(new SignupResponse("회원가입이 완료되었습니다."));
     }
 
+    @PostMapping("/signin/kakao")
+    public ResponseEntity<SigninResponse> kakaoSignin(@Valid @RequestBody KakaoSigninRequest request) {
+        SigninResponse signinResponse = memberService.kakaoSignin(request.getAuthCode());
+
+        if (!"".equals(signinResponse.accessToken)) {
```

Comment:
```suggestion
        if (StringUtils.hasText(signinResponse.accessToken)) {
```

이렇게 작성하는 게 더 가독성이 높을 것 같네요

Reply:
아 맞네요. 말씀하신 것처럼 하는 게 훨씬 가독성 높아 보입니다!